feature/manager.go

```diff
@@ -116,18 +112,6 @@ func newManager(cfg Config, desc setDesc) (*Manager, error) {
 	// Now, remove any features as directed by the config.
 	configFeatures := make(map[Set]*lnwire.FeatureVector)
 	for set, raw := range fsets {
-		if cfg.NoTLVOnion {
```

Comment:
I interpreted @Roasbeef's [comment](https://github.com/lightningnetwork/lnd/pull/8275/commits/0a52d1bcd47b98b9b93d102bf290be8f51b18aac#r1446769182) as leaving this option in and functional. The current iteration of the code basically disables the legacy onion but leaves the command line flags in (but they don't have any effect anymore).

Reply:
Yeah I wondered about this. Thank you for confirming.